### DISCUSSION TITLE
GitHub Copilot: Failed to initiate the GitHub login process. Please try again.

### DISCUSSION THREAD

**Discussion by saraful** [community/community#31182] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/82141016/187115056-ab4d146b-e900-4b87-b9ff-27f0b58d3ffc.png)

IntelliJ IDEA 2021.3.3 (Ultimate Edition)
Copilot 1.1.30.1887
Windows7

Now the old version of copilot can no longer be used. I must update it to the latest version。
But I can't log in now，and I have tried to modify DNS and host，I have also tried to configure HTTP proxy，but it was still no go。
![image](https://user-images.githubusercontent.com/82141016/187116146-2fa5f195-4d0a-471d-930a-857a7b4704f3.png)

What should I do？

有没有吊大的知道咋整，帮帮忙

**Comment by xu-feiran** [comment]
the same problem

**Reply by jar-us** [reply]
same problem for me also. product is not stable, forced to buy licence, even after buying licence now not able to login.

**Comment by paufase** [comment]
Same problem here, solvet it by going back to 1.1.28.1744 (it's the latest version that works)

**Reply by saraful** [reply]
I tried, it still doesn't work.：(

**Comment by zyy-lang** [comment]
how to solve it，He's still charging me

**Reply by saraful** [reply]
I have cancelled my subscription and applied for a refund。I'll subscribe when it's available

**Comment by Moses-Code-Dev** [comment]
Microsoft force users to switch to the latest Windows if you want to use windows 7 or others version, switch to CodeWhisperer From amazon it's a great tools with the same performance as Copilot.

**Comment by Shaojay** [comment]
真操蛋, 买了没卵用
